multiversx_sdk/core/transactions_factories/relayed_transactions_factory_test.py

```diff
@@ -231,3 +232,76 @@ def test_compute_relayed_v2_transaction(self):
         assert relayed_transaction.options == 0
         assert relayed_transaction.gas_limit == 60414500
         assert relayed_transaction.data.decode() == "relayedTxV2@000000000000000000010000000000000000000000000000000000000002ffff@0f@676574436f6e7472616374436f6e666967@fc3ed87a51ee659f937c1a1ed11c1ae677e99629fae9cc289461f033e6514d1a8cfad1144ae9c1b70f28554d196bd6ba1604240c1c1dc19c959e96c1c3b62d0c"
+
+    def test_compute_relayed_v3_transaction(self):
+        alice = self.wallets["alice"]
+        bob = self.wallets["bob"]
+
+        inner_transaction_1 = Transaction(
```

Comment:
can simply be `inner_transaction`

Reply:
changed

Reply:
this is not changed, but it no big deal anyways.